docs/headless/warning-listener-protocol.md

```diff
@@ -0,0 +1,3 @@
+---
+---
+{{< warning >}}After the cluster is created, you cannot change the way the listeners are configured. If a cluster is created with unencrypted (plain text) listener, it cannot be changed later to use SSL encrypted listeners, or the way around.{{< /warning >}}
```

Comment:
I would rephrase this, by saying you can change it but not without outage or manual steps. The manual steps includes killing all the broker pods one by one

Reply:
```suggestion
{{< warning >}}After the cluster is created, you cannot change the way the listeners are configured without an outage. If a cluster is created with unencrypted (plain text) listener and you want to switch to SSL encrypted listeners (or the way around), you must manually delete each broker pod. The operator will restart the pods with the new listener configuration.{{< /warning >}}
```